Formula/php.rb

```diff
@@ -0,0 +1,343 @@
+
+      # Prefer relative symlink instead of absolute for relocatable bottles
+      ln_s "phar.phar", bin+"phar", :force => true if File.exist? bin+"phar.phar"
+
+      # Install new php.ini unless one exists
+      config_path.install "./php.ini-development" => "php.ini" unless File.exist? config_path+"php.ini"
+
+      chmod_R 0775, lib+"php"
+
+      system bin+"pear", "config-set", "php_ini", config_path+"php.ini", "system"
+
+      if File.exist?("sapi/fpm/init.d.php-fpm")
+        chmod 0755, "sapi/fpm/init.d.php-fpm"
+        sbin.install "sapi/fpm/init.d.php-fpm" => "php#{version.to_s[0..2].delete(".")}-fpm"
+      end
+
+      if File.exist?("sapi/cgi/fpm/php-fpm")
+        chmod 0755, "sapi/cgi/fpm/php-fpm"
+        sbin.install "sapi/cgi/fpm/php-fpm" => "php#{version.to_s[0..2].delete(".")}-fpm"
+      end
+
+      if !File.exist?(config_path+"php-fpm.d/www.conf") && File.exist?(config_path+"php-fpm.d/www.conf.default")
+        mv(config_path+"php-fpm.d/www.conf.default", config_path+"php-fpm.d/www.conf")
+      end
+
+      unless File.exist?(config_path+"php-fpm.conf")
+        if File.exist?("sapi/fpm/php-fpm.conf")
+          config_path.install "sapi/fpm/php-fpm.conf"
+        end
+
+        if File.exist?("sapi/cgi/fpm/php-fpm.conf")
+          config_path.install "sapi/cgi/fpm/php-fpm.conf"
+        end
+
+        inreplace config_path+"php-fpm.conf" do |s|
+          s.sub!(/^;?daemonize\s*=.+$/, "daemonize = no")
+        end
+      end
+
+      rm_f("#{config_pear}-backup") if File.exist? "#{config_pear}-backup"
+      rm_f("#{user_pear}-backup") if File.exist? "#{user_pear}-backup"
+      rm_f("#{config_pearrc}-backup") if File.exist? "#{config_pearrc}-backup"
+      rm_f("#{user_pearrc}-backup") if File.exist? "#{user_pearrc}-backup"
+    rescue StandardError
+      mv("#{config_pear}-backup", config_pear) if File.exist? "#{config_pear}-backup"
+      mv("#{user_pear}-backup", user_pear) if File.exist? "#{user_pear}-backup"
+      mv("#{config_pearrc}-backup", config_pearrc) if File.exist? "#{config_pearrc}-backup"
+      mv("#{user_pearrc}-backup", user_pearrc) if File.exist? "#{user_pearrc}-backup"
+      raise
+    end
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{HOMEBREW_PREFIX}/opt/php#{version.to_s[0..2].delete(".")}/libexec/apache2/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini file can be found in:
+          #{etc}/php/#{version.to_s[0..2]}/php.ini
+    EOS
+
+    if build.with? "pear"
+      s << <<-EOS.undent
+        ✩✩✩✩ PEAR ✩✩✩✩
+
+        If PEAR complains about permissions, 'fix' the default PEAR permissions and config:
+
```

Comment:
I will do it if I'll keep the caveat. The permissions required can be fixed in the Formula so there should be no need to do this really.